onedrivesdk/src/main/java/com/onedrive/sdk/generated/BaseDeltaCollectionPage.java

```diff
@@ -40,7 +40,7 @@
 /**
  * The class for the Base Delta Collection Page.
  */
-public class BaseDeltaCollectionPage extends BaseCollectionPage<Item, IDeltaRequestBuilder> implements IBaseDeltaCollectionPage {
+public class BaseDeltaCollectionPage extends BaseCollectionPage<DeltaResponse, IDeltaRequestBuilder> implements IBaseDeltaCollectionPage {
```

Comment:
probably we want to do it for a separated one as the model generator has been changed?